.github/workflows/ftr-e2e-dashboards-observability-test.yml

```diff
@@ -0,0 +1,163 @@
+name: FTR E2E Dashboards observability Test
+
+on: [pull_request, push]
```

Comment:
Can we run it nightly as well? To spot flaky tests.

```
on:
  schedule:
    - cron: '0 0 * * *'
```

Reply:
yes lets do that

Reply:
I just enabled. Good call.